azure-service-bus-management/1.0/modules/ROOT/pages/azure-service-bus-management-connector-examples.adoc

```diff
@@ -0,0 +1,246 @@
+= Azure Service Bus Management 1.0 Examples - Mule 4
+
+. Declare and complete the variables previously described in the corresponding YAML file (for example: `src/main/resources/properties/kafka-producer.yaml`), and load that properties file as a global element.
++
+For more information about using a properties file, see xref:mule-runtime::mule-app-properties-to-configure.adoc[Configuring Property Placeholders]
+
+
+You can check more use cases on the demo `mule-azure-service-bus-management-crud-demo` .
```

Comment:
Not sure these demos are available. You and I both asked Mauro about this. I don't think users can access Mule 4 demos.